dp-hello-world-controller/ci/scripts/lint.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash -eux
 
+go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.59.1
```

Comment:
Should this be a later linter version given it's now at v1.63.4 and the tag is v1.62 in the lint.yml file?